compiler/AST/expr.cpp

```diff
@@ -5639,11 +5546,7 @@ GenRet CallExpr::codegen() {
 
     // Handle passing strings to externs
     if (fn->hasFlag(FLAG_EXTERN)) {
-      if (actualType == dtString)
-        arg = codegenCastToCharStar(codegenValue(arg));
-      else if (isRefWideString(actualType))// checks for ref(widestr)
-        arg = codegenAddrOf(codegenWideThingField(codegenDeref(arg),WIDE_GEP_ADDR));
-      else if( actualType->symbol->hasFlag(FLAG_WIDE_REF) ||
```

Comment:
Does this mean that you must use .c_str() when passing a string to an extern?
Or are we automatically coercing strings to c_string elsewhere?